service.subtitles.subdivx/service.py

```diff
@@ -0,0 +1,738 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd recommend to move all your logic into a separate importable module leaving only a small portion of code in the addon entrypoint. This will allow to reduce addon start time on systems with limited CPU resources like Raspberry Pi or similar.